### PR TITLE
feat: 96/abort session and clear steps

### LINKED ISSUE #96: feat: update play/pause/stop functionality per new design
Related to https://github.com/elastic/synthetics-recorder/issues/70.

Per designs in the linked meta issue, we should implement a new flow for play/pause/stop in the recorder's flow.

- [x] Start button will now become Pause during a run
- [x] No Stop button, stop button is now "start over", which will re-launch a new session.
  - for now, this will kill the existing chromium process, and focus on the URL input. Eventually we will have a "start" page that consists of just the URL input and a start button.

More info/design on the parent issue.

### PR DESCRIPTION
**Pull request by justinkambic**
**NOTE TO REVIEWERS:** Depends on https://github.com/elastic/synthetics-recorder/pull/114.

Resolves #96.

This is a reposting of https://github.com/elastic/synthetics-recorder/pull/97; after we brought this repo public my fork lost association with open PRs and new commits would not show up/trigger the CI process.

After numerous attempts to re-establish the connection I opted to simply re-open a new PR. Description and previous comments available at the link above.

### CODE REVIEW

**Review by vigneshshanmugam — CHANGES_REQUESTED** [review]
I am confused with some of the changes in this PR.

1. Start over does not create a new browser session.
2. I can create endless no of browser sessions with Start recording and Start over. Previous browser sessions are not closed properly.
3. Previous actions are still showing up even if I start over.

**Comment by justinkambic** [comment]
@vigneshshanmugam sorry, I should've run this branch before re-opening the PR. The functionality is not working the way it was when I last developed on this branch, I will ping you when it's ready for review.

**Comment by justinkambic** [comment]
@vigneshshanmugam I think this is now working as the design expects.

**Review by vigneshshanmugam — APPROVED** [review]
LGTM pending a small change.

File: src/components/Header.tsx [hunk 1]
```diff
@@ -58,28 +76,33 @@ export function Header(props: IHeader) {
...
         </EuiFlexItem>
         <EuiFlexItem grow={false}>
           <ControlButton
-            aria-label="Toggle script recording on/off"
+            aria-label="Toggle the script recorder between recording and paused"
             fill
             color="primary"
-            iconType={isRecording ? "stop" : "play"}
-            onClick={toggleRecording}
+            iconType={isRecording ? "pause" : "play"}
```
[the remark this answers is not in the document]
Comment (justinkambic) [reply src/components/Header.tsx:87]: I'm happy to do that, sure!

File: package.json [hunk 2]
```diff
@@ -82,6 +82,7 @@
     "husky": "^7.0.4",
     "jest": "^26.6.0",
     "prettier": "2.5.0",
+    "nodemon": "^2.0.15",
```
[the remark this answers is not in the document]
Comment (justinkambic) [reply package.json:85]: must be, will debug.

[the remark this answers is not in the document]
Comment (justinkambic) [reply package.json:85]: sorry that you've run into several of these recently.

[the remark this answers is not in the document]
Comment (justinkambic) [reply package.json]: FYI a rebase of latest `main` seems to have cleaned up that diff.

File: src/components/Header.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (justinkambic) [reply src/components/Header.tsx]: I've added an `enum` implementation here: https://github.com/elastic/synthetics-recorder/pull/105/commits/2349d6fe85b861a6aeabf52c1f2d1e204335f6ba

LMK if that is ok to you.

[the remark this answers is not in the document]
Comment (vigneshshanmugam) [reply src/components/Header.tsx]: Yes looks good, thanks 👍🏽

**Review by vigneshshanmugam — APPROVED** [review]

**Pull request merged**